src/test/java/uk/gov/hmcts/reform/jobscheduler/services/jobs/jobsservice/CreateJobTest.java

```diff
@@ -0,0 +1,105 @@
+package uk.gov.hmcts.reform.jobscheduler.services.jobs.jobsservice;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.quartz.JobDetail;
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.quartz.Trigger;
+import uk.gov.hmcts.reform.jobscheduler.jobs.HttpCallJob;
+import uk.gov.hmcts.reform.jobscheduler.model.Job;
+import uk.gov.hmcts.reform.jobscheduler.services.jobs.ActionSerializer;
+import uk.gov.hmcts.reform.jobscheduler.services.jobs.JobException;
+import uk.gov.hmcts.reform.jobscheduler.services.jobs.JobsService;
+
+import java.sql.Date;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.jobscheduler.SampleData.validJob;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CreateJobTest {
```

Comment:
I like these tests. Worth adding a test or two for ActionSerializer, as well.

Reply:
done